modules/virt-querying-metrics.adoc

```diff
@@ -66,13 +66,35 @@ Returns the total amount (in bytes) of the virtual machine's storage-related tra
 `kubevirt_vmi_storage_write_traffic_bytes_total`::
 Returns the total amount of storage writes (in bytes) of the virtual machine's storage-related traffic.
 
-.Example storage-related traffic query
+.Examples of storage-related traffic queries
```

Comment:
Nit: To align with the other single examples in this file, I think you should revert this line to `.Example storage-related traffic query`

Reply:
Done.